pkg/persistence/migrator.go

```diff
@@ -118,6 +118,20 @@ CREATE TABLE IF NOT EXISTS tek_upload_count (
 	first_upload		BOOLEAN					NOT NULL DEFAULT FALSE,
 	INDEX (originator),
 	INDEX (date)
+)`,
+		},
+	},{
+		id: "9",
+		statements: []string{
+			`
+CREATE TABLE IF NOT EXISTS otk_life_duration (
+	originator	VARCHAR(32) 	NOT NULL,
+	hours		INT				NOT NULL,
+	date		DATE			NOT NULL,
+	count		INT				UNSIGNED NOT NULL DEFAULT 0,
```

Comment:
Not sure what the formatter is doing here

Reply:
I hate the go tabs, prefer 2 spaces, I can fix

Reply:
well I'll fix later.

Reply:
in a seperate PR